### DISCUSSION TITLE
Runtime Error "Access to the path [...] is denied"

### DISCUSSION THREAD

**Discussion by AnmolKmb** [SafeExamBrowser/seb-win-refactoring#343] [Q&A]
I've installed the SAFE Exam Browser but upon running the program, "Fatal Error" pops up. I don't know what might be the issue but when I try running it as admin, it runs good. Afterwards, since the configuration file isn't synced, the exam can't be given by me. I've tried a lot of times at different locations and even tried reinstalling, restarting but it all bore no fruit.

Device specs are:
Laptop:- Dell Inspiron 15 3000
OS :- Windows 10

I'm also attaching the screenshot of the error for your convenience. Please do revert back to me ASAP as I've got academic commitments to fulfill thereafter.

![safe](https://user-images.githubusercontent.com/98400806/150985271-7a8fe188-a4b3-4e5d-a53a-c1755259078b.png)

**Comment by dbuechel** [comment]
As the error already indicates, SEB can't access the directory specified in the message. Please make sure that you installed SEB for the user with which you intend to use SEB, and furthermore verify that this particular user has the necessary access rights for the directory which SEB currently can't access.
